src/cmake/dependencies.cmake

```diff
@@ -31,6 +31,8 @@ else()
   set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${DEPS_DIR}/eigen-git-mirror/cmake")
 endif()
 
+set(IMGUI_DIR "${DEPS_DIR}/imgui")
```

Comment:
can you wrap this in `if(NOT IMGUI_DIR)` so it works also with my system-wide magnum installation? :)